File: TAREA3.java
import java.util.ArrayList;

public class Main {
    public static void main(String[] args) {
        ArrayList<Integer> arr = new ArrayList<>();
        arr.add(10);
        arr.add(20);
        arr.add(30);

        // Insertar elemento
        arr.add(40);

        // Recorrer e imprimir
        System.out.println("Elementos del arreglo:");
        for (int x : arr) {
            System.out.println(x);
        }
    }
}

File: TAREA3.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    vector<int> arr = {10, 20, 30};

    // Insertar elemento
    arr.push_back(40);

    // Recorrer e imprimir
    cout << "Elementos del arreglo:\n";
    for (int x : arr) {
        cout << x << endl;
    }

    return 0;
}